### ISSUE TITLE
dash_table was not found

### ISSUE THREAD

**Issue by mangel2095** [plotly/dash-table#496]
Today I update plotly and when I run a code that use a dash_table it looks like this:

![imagen](https://user-images.githubusercontent.com/18307144/61022765-df520280-a36c-11e9-8c07-0871c9937228.png)

I thought maybe the problem was my code, so I tried with the first code example from this page:
https://dash.plot.ly/datatable/interactivity
And the same thing happens. 

I use a w7 64 bits.
pyplot version:
![imagen](https://user-images.githubusercontent.com/18307144/61022978-a7978a80-a36d-11e9-88af-8755a35da5d8.png)
dash version:
![imagen](https://user-images.githubusercontent.com/18307144/61023022-cbf36700-a36d-11e9-9803-b8c1028c9cb8.png)

Thanks in advance for the help!

**Comment by thehapax** [comment]
Yes, can someone please fix this. I have the same settings as @mangel2095 here and I copied the very simple example from the readme (https://github.com/plotly/dash-table)  and get the error:

```
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/dash/dash.py", line 1133, in dispatch
    for component_registration in self.callback_map[output]['inputs']:
KeyError: 'live-update-text.children'```

**Comment by thehapax** [comment]
Further if i modify the code to use a url, as I did here: 

```
import dash
import dash_table
import pandas as pd

df = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/gapminder2007.csv')

app = dash.Dash(__name__)

app.layout = dash_table.DataTable(
    id='table',
    columns=[{"name": i, "id": i} for i in df.columns],
    data=df.to_dict("rows"),
)

if __name__ == '__main__':
    app.run_server(debug=True)
```


 i keep getting the following SSL error: 

```  
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1045)>
```

**Comment by thehapax** [comment]
> Yes, can someone please fix this. I have the same settings as @mangel2095 here and I copied the very simple example from the readme (https://github.com/plotly/dash-table) and get the error:

OK, so I discovered, stopping the server and reloading the above fixes the problem for the local CSV. 

For the SSL certificate error on macs, this was the fix: 

https://timonweb.com/tutorials/fixing-certificate_verify_failed-error-when-trying-requests_html-out-on-mac/

**Comment by thehapax** [comment]
[SOLVED] 

I had to uninstall all of the dash packages and install.  It appears the older versions of dash and related packages are not compatible with dash_table. The following now works: 

```
plotly                    3.10.0  
dash                      1.0.1       
dash-core-components      1.0.0       
dash-daq                  0.1.0       
dash-html-components      1.0.0       
dash-renderer             1.0.0       
dash-table                4.0.1     
```

**Comment by Marc-Andre-Rivet** [comment]
@thehapax @mangel2095 I did not notice the version discrepancy when I read through the issue last week, sorry about that.

So yes, as of dash==0.37.0 (https://github.com/plotly/dash/blob/master/dash/CHANGELOG.md#0370---2019-02-11), the core packages (dash-core-components, dash-html-components, dash-renderer, dash-table) and dash are tied together at a specific version. While some unsupported combinations might sometimes work, there's no guarantee they will or when they will break.

Closing as solved.
